index.md

```diff
@@ -50,7 +50,6 @@ PRs against [rust-lang/rust-forge].
 ### Other things
 
 * [Friend of the Tree archives](fott.html).
-* [Bibliography](bibliography.html). Research papers and other links to projects that influenced Rust. Papers about Rust.
```

Comment:
So, let's link this to rustc-guide bibliography and add a redirect from the forge bibliography to the rustc-guide one.